src/multiselect.js

```diff
@@ -243,6 +295,7 @@ export class Multiselect {
    * @private
    */
   eventListener_(e) {
+    console.log(e)
```

Comment:
You should remove this and the other `console.log` statements that you added for debugging/testing.